plugins/linux-capture/pipewire.c

```diff
@@ -439,17 +439,21 @@ static uint32_t build_format_params(obs_pipewire_data *obs_pw,
 	params =
 		bzalloc(2 * obs_pw->format_info.num * sizeof(struct spa_pod *));
 
+	if (!check_pw_version(&obs_pw->server_version, 0, 3, 33))
+		goto build_shm;
+
 	for (size_t i = 0; i < obs_pw->format_info.num; i++) {
-		if (obs_pw->format_info.array[i].modifiers.num == 0 ||
-		    !check_pw_version(&obs_pw->server_version, 0, 3, 33)) {
+		if (obs_pw->format_info.array[i].modifiers.num == 0) {
 			continue;
 		}
+
```

Comment:
Nitpick: this change seems unrelated to this commit, could you please merge it into the commit you originally introduced this code?